packages/scene-composer/src/components/SceneComposerInternal.spec.tsx

```diff
@@ -1,11 +1,16 @@
 import * as React from 'react';
 import renderer, { act } from 'react-test-renderer';
 import str2ab from 'string-to-arraybuffer';
+import flushPromises from 'flush-promises';
 
 import { SceneComposerInternal, SceneComposerApi, useSceneComposerApi } from '..';
 import * as SceneLayoutComponents from '../layouts/SceneLayout';
 import { invalidTestScenes, testScenes } from '../../tests/testData';
 
+jest.mock('@awsui/components-react', () => ({
+  ...jest.requireActual('@awsui/components-react'),
+}));
+
```

Comment:
We are trying to move away from needing to use the real aws-ui in tests, not introduce new cases of this.